.github/labels.yml

```diff
@@ -0,0 +1,49 @@
+# Main categories
+- name: 'breaking'
+  color: 'b60205'
+  description: 'Breaking API change or removal of functionality'
```

Comment:
I think this would be a supplementary tag, like an adjective. 

So "This card is a feature, and it is breaking" or "This is a bugfix, and it is breaking".

Reply:
@amostergaard My own opinion but I still think it's unclear. Would it be different from having a supplementary tag like "Critical"?

Reply:
For sure @camthorsen  - so breaking does have a technical definition in our community. It indicates that there will be an incompatibility between this and version and previous versions. 

Having this tag would be important for things, like, flagging that a pull request would cause a major version change.